analytics/gunicorn.conf.py

```diff
@@ -1,2 +1,7 @@
 bind = ["0.0.0.0:8090"]
+capture_output = True
+chdir = "./analytics/"
+timeout = 120
+reload = True
+loglevel = "debug"
```

Comment:
Should this be configured by environment?

Reply:
It could be. But currently all Gunicorn configs are set to be log-level debug.